hopefully fix build on php5.1 (though I am not sure about the content.c part)

--- eaccelerator-0.9.3/content.c~	2005-04-27 22:39:18.000000000 +0200
+++ eaccelerator-0.9.3/content.c	2005-12-24 16:37:03.468541456 +0100
@@ -593,10 +593,12 @@
     if (zend_eval_string(code, return_value, eval_name TSRMLS_CC) == SUCCESS &&
         eaccelerator_content_cache_place != eaccelerator_none) {
 
+#if 0
       /* clean garbage */
       while (EG(garbage_ptr)) {
         zval_ptr_dtor(&EG(garbage)[--EG(garbage_ptr)]);
       }
+#endif
 
       eaccelerator_put(key, key_len, return_value, ttl, eaccelerator_content_cache_place TSRMLS_CC);
     }
--- eaccelerator-0.9.3/optimize.c~	2005-04-26 17:04:34.000000000 +0200
+++ eaccelerator-0.9.3/optimize.c	2005-12-24 16:40:37.332029264 +0100
@@ -250,14 +250,18 @@
                }
                break;
              case ZEND_UNSET_VAR:
+#ifdef ZEND_UNSET_DIM_OBJ
              case ZEND_UNSET_DIM_OBJ:
+#endif
                end->result.op_type = IS_UNUSED;
                break;
              case ZEND_RECV:
              case ZEND_RECV_INIT:
              /*case ZEND_ADD_ARRAY_ELEMENT:*/
              case ZEND_INCLUDE_OR_EVAL:
+#ifdef ZEND_JMP_NO_CTOR
              case ZEND_JMP_NO_CTOR:
+#endif
              case ZEND_FE_FETCH:
 #ifdef ZEND_ENGINE_2
              case ZEND_ASSIGN_DIM:
@@ -1279,7 +1283,9 @@
               goto jmp_nz;
             }
             goto jmp_2;
+#ifdef ZEND_JMP_NO_CTOR
           case ZEND_JMP_NO_CTOR:
+#endif
           case ZEND_FE_FETCH:
 jmp_2:
             while (p->jmp_2->len == 1 && p->jmp_2->start->opcode == ZEND_JMP) {
@@ -2437,7 +2443,9 @@
           op->opcode == ZEND_DO_FCALL_BY_NAME ||
           op->opcode == ZEND_POST_INC ||
           op->opcode == ZEND_POST_DEC ||
+#ifdef ZEND_UNSET_DIM_OBJ
           op->opcode == ZEND_UNSET_DIM_OBJ ||
+#endif
           op->opcode == ZEND_INCLUDE_OR_EVAL
 #ifdef ZEND_ENGINE_2
           || op->opcode == ZEND_ASSIGN_DIM
@@ -2715,7 +2723,9 @@
 			case ZEND_JMPNZ:
 			case ZEND_JMPZ_EX:
 			case ZEND_JMPNZ_EX:
+#ifdef ZEND_JMP_NO_CTOR
 			case ZEND_JMP_NO_CTOR:
+#endif
 			case ZEND_FE_FETCH:
 				bb[line_num+1].start = op+1;
 				bb[op->op2.u.opline_num].start = &op_array->opcodes[op->op2.u.opline_num];
@@ -2834,7 +2844,9 @@
 				break;
 #endif
 			case ZEND_UNSET_VAR:
+#ifdef ZEND_UNSET_DIM_OBJ
 			case ZEND_UNSET_DIM_OBJ:
+#endif
 				op->result.op_type = IS_UNUSED;
 				break;
 			default:
@@ -2888,7 +2900,9 @@
 				case ZEND_JMPNZ:
 				case ZEND_JMPZ_EX:
 				case ZEND_JMPNZ_EX:
+#ifdef ZEND_JMP_NO_CTOR
 				case ZEND_JMP_NO_CTOR:
+#endif
 				case ZEND_FE_FETCH:
 					p->jmp_2 = &bb[op->op2.u.opline_num];
 					p->follow = &bb[line_num];
